temporal/t.support/t.support.html

```diff
@@ -40,9 +40,3 @@ <h2>SEE ALSO</h2>
 <h2>AUTHOR</h2>
 
 S&ouml;ren Gebbert, Th&uuml;nen Institute of Climate-Smart Agriculture
-
-<!--
-<p>
-<i>Last changed: $Date$</i>
--->
-
```

Comment:
`<h2>Examples</h2>` should be `<h2>EXAMPLES</h2>`.

Line 32 unnecessary one empty line.